src/metrics/extract_ttfs.py

```diff
@@ -90,9 +91,9 @@ def main():
             height = depth = italic = skew = width = 0
             glyph = glyf[name]
             if glyph.numberOfContours:
-                height = glyph.yMax
-                depth = -glyph.yMin
-                width = glyph.xMax - glyph.xMin
+                height = glyph.yMax / unitsPerEm
+                depth = -glyph.yMin / unitsPerEm
+            width = widths[name].width / unitsPerEm
```

Comment:
You can get width by `glyph.width`.

Reply:
No, I tried that, and it wasn't defined. Unless I somehow have the wrong version installed? I did it pretty recently.

Reply:
@edemaine Ah, sorry, I was looking at `fontforge` Python API.